### PR TITLE
Dmirano/4510 state staff and expenses help text

### LINKED ISSUE #4510: [Feature] Help Text Changes for State Staff and Expenses
### Description and related issues
Based on design ticket #4381 

### Mockups or link to Figma
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=3328%3A18573&t=VXupikAaT7lnGE7V-1

State Staff

Other state expenses

### Acceptance criteria

- [ ] change State Staff example box to
> **Example:** Two Auditors, Class 1, working at 75% time = Auditor, Class 1 at 1.5 FTE
>
> **Example:** Two Auditors, Class 1 and Class 2, working at 75% time = Auditor, Class 1, 0.75 FTE and Auditor, Class 2, 0.75 FTE.

- [ ] change Other State Expenses example box to 
> Refer to SMDL 16-004 for guidance on allowable expenses and FFP rates.

### PR DESCRIPTION
**Pull request by mirano-darren**
Resolves #4510 

### Description
Updates help text for the mmis state staff and expenses page

### Significant changes or possible side effects
None, all changes isolated to state staff and expenses page

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |

### Steps to manually verify this change

1. Open a HITECH APD and verify the state staff and expenses has the old help text
2. Open a MMIS APD and verify the state staff and expenses has the new help text

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] Associated OpenAPI documentation has been updated
- [ ] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/pages/apd/activities/state-costs/NonPersonnelCosts.test.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ describe('activity non-personnel costs subsection', () => {
         },
         { activityIndex: 2 }
       )
-    ).toEqual({ expenses: 'these are expenses' });
+    ).toEqual({ expenses: 'these are expenses', apdType: 'HITECH' });
```
Comment (thetif) [inline web/src/pages/apd/activities/state-costs/NonPersonnelCosts.test.js:69]: you should use the APD_TYPE.HITECH const here

File: web/src/pages/apd/activities/state-costs/StatePersonnel.test.js [hunk 2]
```diff
@@ -61,7 +61,7 @@ describe('activity state personnel costs subsection', () => {
         },
         { activityIndex: 0 }
       )
-    ).toEqual({ personnel: 'these are personnel' });
+    ).toEqual({ personnel: 'these are personnel', apdType: 'HITECH' });
```
Comment (thetif) [inline web/src/pages/apd/activities/state-costs/StatePersonnel.test.js:64]: you should use the APD_TYPE.HITECH const here

**Review by thetif — APPROVED** [review]

**Review by jeromeleecms — CHANGES_REQUESTED** [review]
@mirano-darren For the link to the SMDL letter, can you adjust it to open in a new tab instead of in the same window. This is the case for other external link through out the app (and you can't just back navigate to the APD).

**Review by jeromeleecms — APPROVED** [review]
Great - thanks for the quick fix. Good to go on my end.

**Review by SGilliamA1M — APPROVED** [review]
Verified that a HITECH APD has the old help text and MMIS APD has the new help text. 
Well done!

**Pull request merged**